jaxrs/src/main/java/org/killbill/billing/jaxrs/json/CatalogJson.java

```diff
@@ -375,22 +426,204 @@ public int hashCode() {
         }
     }
 
+    public static class TieredBlockJson {
```

Comment:
@pierre 
Using JDK7 I was able to generate the files!
Do I have to do anything else? Push any change?
I've checked that the new JAX-RS models are now generated in Ruby.

Thanks!